### PR TITLE
Enable PHP 7.4 snapshot

### PR DESCRIPTION
**Pull request by sagikazarmark**

### CODE REVIEW

**Comment by Tobion** [comment]
build failed

**Comment by sagikazarmark** [comment]
Looks like we are kinda forced to drop PHP 5.4 and 5.5

**Comment by Tobion** [comment]
Symfony 3.4 runs on php 5.5 as well. So this should be possible.

**Comment by sagikazarmark** [comment]
Looks like it doesn't: https://travis-ci.org/symfony/symfony/jobs/568036831

**Comment by gmponos** [comment]
It is also breaking the build on Guzzle Client [check](https://travis-ci.org/guzzle/guzzle/builds/567294254?utm_source=github_status&utm_medium=notification)

**Comment by sagikazarmark** [comment]
It's not the tests that don't pass. Travis fails to start with PHP 5.4 and 5.5. I'm okay with removing the last commit from the HEAD, but I have no idea why suddenly those PHP versions stopped working on Travis. Check the broken builds.

**Comment by sagikazarmark** [comment]
Looks like it might be a distribution issue.

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 language: php
 
+# See https://travis-ci.community/t/php-5-4-and-5-5-archives-missing/3723
+dist: trusty
```
Comment (Tobion) [inline .travis.yml]: we only need trust on 5.4 and 5.5

File: .travis.yml [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
     - 7.3
+    - 7.4snapshot
     - hhvm
 
 matrix:
+    # See https://travis-ci.community/t/php-5-4-and-5-5-archives-missing/3723
+    include:
+        - php: 5.4
+          dist: trusty
+        - php: 5.4
```
Comment (Tobion) [inline .travis.yml]: you have php 5.4 twice...

**Comment by sagikazarmark** [comment]
Grrrrr, it's hard to make you happy @Tobion 😂

**Pull request merged**